src/coreclr/jit/emitxarch.cpp

```diff
@@ -7622,7 +7622,10 @@ void emitter::emitIns_R_AR(instruction ins, emitAttr attr, regNumber reg, regNum
     emitIns_R_ARX(ins, attr, reg, base, REG_NA, 1, disp);
 }
 
-void emitter::emitIns_R_AI(instruction ins, emitAttr attr, regNumber ireg, ssize_t disp)
+void emitter::emitIns_R_AI(instruction ins,
```

Comment:
We have a few various `emitIns_*A*` methods, do any of the others need the same handling?

Is this something we can/should centralize somewhere?

Reply:
Probably? If I find another case that doesn't render comments I'll check what we can do to centralize it